dc-commons-jdbi/CHANGELOG.md

```diff
@@ -0,0 +1,14 @@
+# Changelog
+
+All notable changes to this project will be documented in this file.
+
+The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/) and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0.html).
+
+## [Unreleased]
+
+## [4.2.3](https://github.com/dbmdz/digitalcollections-commons/releases/tag/dc-commons-jdbi-4.2.3) - 2022-04-12
+
+### Added
+
+- **Breaking**: bumped `dc-model` dependency to version `9.2.0`
```

Comment:
Wieso ist das ein breaking change? Die vorherige Version (in dc-commons-jdbi 4.2.2) war 9.1.1, und in 9.2.0 sind keine breaking changes gekommen.

Reply:
Danke für deinen Einwand, du hast vollkommen recht :+1: